urumanifest/utils.py

```diff
@@ -20,7 +20,7 @@
 
 def check_python_version(py_exe, py_version=(2,7)):
     logging.debug(f"Checking Python interpreter version: {py_exe}")
-    if not py_exe or not py_exe.is_file():
+    if not py_exe:
```

Comment:
?

Reply:
This should not have gone in. I was getting an exception here and it was quicker just to disable this check. I'll put it back and then we can talk over the issue.